servant-client/src/Servant/Client.hs

```diff
@@ -417,6 +417,12 @@ instance HasClient api => HasClient (IsSecure :> api) where
   clientWithRoute Proxy req baseurl manager =
     clientWithRoute (Proxy :: Proxy api) req baseurl manager
 
+instance HasClient subapi =>
+  HasClient (WithNamedConfig name config subapi) where
```

Comment:
Doesn't there need to be an instance for `(WithNamedConfig name config s) :> sublayout` too?